### PR TITLE
Logs show page

### PR DESCRIPTION
**Pull request by FelixMalfait**
Basic implementation of logs that's fully functional now but without CSS

It's behind a feature flag so acceptable to merge it like this

Screenshot 2024-03-22 at 09 43 21

### CODE REVIEW

File: packages/twenty-server/src/engine/api/graphql/workspace-query-runner/jobs/save-event-to-db.job.ts [hunk 1]
```diff
@@ -33,22 +34,35 @@ export class SaveEventToDbJob implements MessageQueueJob<SaveEventToDbJobData> {
...
       return;
     }
 
+    if (data.details.diff) {
+      data.details = {
+        diff: data.details.diff,
+      };
+    }
+
+    const workspaceMember = await workspaceDataSource?.query(`
```
Comment (Weiko) [inline packages/twenty-server/src/engine/api/graphql/workspace-query-runner/jobs/save-event-to-db.job.ts:53]: Using prepared param
```suggestion
    const workspaceMember = await workspaceDataSource?.query(`
          SELECT "id" FROM ${dataSourceMetadata.schema}."workspaceMember"
      WHERE ("userId" = $1) LIMIT 1;
      `, [data.userId])
```

Comment (Weiko) [reply packages/twenty-server/src/engine/api/graphql/workspace-query-runner/jobs/save-event-to-db.job.ts:53]: Also you can now move this logic to repositories inside `packages/twenty-server/src/modules/workspace-member/repositories/workspace-member.repository.ts`
To inject it 
```
    @InjectObjectMetadataRepository(WorkspaceMemberObjectMetadata)
    private readonly workspaceMemberService: WorkspaceMemberRepository,
```
then import this module
```
ObjectMetadataRepositoryModule.forFeature([
      WorkspaceMemberObjectMetadata,
    ]),
```

File: packages/twenty-server/src/engine/api/graphql/workspace-query-runner/listeners/entity-events-to-db.listener.ts [hunk 2]
```diff
@@ -58,6 +65,7 @@ export class EntityEventsToDbListener {
 
     this.messageQueueService.add<SaveEventToDbJobData>(SaveEventToDbJob.name, {
       workspaceId: payload.workspaceId,
+      userId: payload.userId,
```
Comment (Weiko) [inline packages/twenty-server/src/engine/api/graphql/workspace-query-runner/listeners/entity-events-to-db.listener.ts:68]: Shouldn't we use the workspaceMemberId instead?

File: packages/twenty-server/src/engine/integrations/event-emitter/utils/object-record-changed-values.ts [hunk 3]
```diff
@@ -0,0 +1,28 @@
+import deepEqual from 'deep-equal';
+
+export const objectRecordChangedValues = (
```
Comment (Weiko) [inline packages/twenty-server/src/engine/integrations/event-emitter/utils/object-record-changed-values.ts:3]: Can we add a test to this util?

File: packages/twenty-server/src/engine/api/graphql/workspace-query-runner/listeners/entity-events-to-db.listener.ts (hunk 2 above)
Comment (FelixMalfait) [reply packages/twenty-server/src/engine/api/graphql/workspace-query-runner/listeners/entity-events-to-db.listener.ts:68]: We don't have access to the workspaceMemberId easy in that context, that's why I retrieve it later

**Review by Weiko — APPROVED** [review]
LGTM

**Pull request merged**